From 140d53f558df6a50b754a4ade4e35113a0303af9 Mon Sep 17 00:00:00 2001
From: Jeegar Patel <jeegar.patel@xilinx.com>
Date: Tue, 22 Sep 2020 05:05:27 -0700
Subject: [PATCH 1624/1852] drm: xlnx: bridge: Fix PW.PARAMETER_HIDDEN type
 warning

commit 4606cbad1a98d334f8ea45612a3ab08324f8451f from
https://github.com/Xilinx/linux-xlnx.git

This patch fix PW.PARAMETER_HIDDEN type coverity warning.

Event parameter_hidden: declaration hides parameter "f"
(declared at line 331)

Signed-off-by: Jeegar Patel <jeegar.patel@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_bridge.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_bridge.c b/drivers/gpu/drm/xlnx/xlnx_bridge.c
index 7015beba3e6a..016cfcf6fb07 100644
--- a/drivers/gpu/drm/xlnx/xlnx_bridge.c
+++ b/drivers/gpu/drm/xlnx/xlnx_bridge.c
@@ -342,7 +342,7 @@ static ssize_t xlnx_bridge_debugfs_write(struct file *f, const char __user *buf,
 		xlnx_bridge_disable(bridge);
 	} else if (!strncmp(buf, "set_input", 3)) {
 		char *cmd, **tmp;
-		char *w, *h, *f;
+		char *w, *h, *bus_fmt;
 		u32 width = 0, height = 0, fmt = 0;
 		int ret;
 
@@ -359,11 +359,11 @@ static ssize_t xlnx_bridge_debugfs_write(struct file *f, const char __user *buf,
 		strsep(tmp, " ");
 		w = strsep(tmp, " ");
 		h = strsep(tmp, " ");
-		f = strsep(tmp, " ");
-		if (w && h && f) {
+		bus_fmt = strsep(tmp, " ");
+		if (w && h && bus_fmt) {
 			ret = kstrtouint(w, 0, &width);
 			ret |= kstrtouint(h, 0, &height);
-			ret |= kstrtouint(f, 0, &fmt);
+			ret |= kstrtouint(bus_fmt, 0, &fmt);
 		}
 
 		kfree(cmd);
-- 
2.31.1

